### PR TITLE
Remove code tag

### PR DESCRIPTION
**Pull request by TobiasSommer**
Updated the code-Tag to the literal-Tag

### CODE REVIEW

File: en/content/administration/email/email.xml [hunk 1]
```diff
@@ -114,6 +114,32 @@ If the email is shown in the QueueView, then your setup is working.
 
 <example id='procmail-queue-routing'>
 
+<<<<<<< HEAD
```
Comment (mgruner) [inline en/content/administration/email/email.xml]: This looks broken. Probably you didn't resolve a merge conflict?

**Comment by TobiasSommer** [comment]
Did a hard reset and solved the merge conflict, I hope.

**Comment by mgruner** [comment]
Seems to be still here as there are no new commits, can you please check? We're close! :+1:

**Comment by mgruner** [comment]
Fixed in #39.

**Pull request closed without merging**